build.gradle

```diff
@@ -9,6 +12,11 @@ Base directory: $projectDir
 Running script ${relativePath(buildFile)}
 """
 
+if (file("vendor/protobuf").list().length == 0) {
+	throw new GradleException(
```

Comment:
Sounds good, I will fix the line space and look at the coding style in another PR.